EIPS/eip-IMPERSONATECALL.md

```diff
@@ -0,0 +1,94 @@
+---
+eip: <undefined number>
+title: IMPERSONATECALL Opcode
+author: Sergio Demian Lerner (sergio.d.lerner@gmail.com)
+category: Core
+type: Standards Track
+status: Draft
+created: 2020-09-24
+---
+
+### Overview
+
+Add a new opcode, `IMPERSONATECALL` at `0xf6`, which is similar in idea to `CALL`, except that it impersonates a sender, i.e. the callee sees a sender different from the real caller. To prevent collisions with other deployed contract or externally owned accounts, the impersonated sender address is derived from the real caller address and a salt.
```

Comment:
```suggestion
Add a new opcode, `IMPERSONATECALL` at `0xf6`, which is similar in idea to `CALL (0xF1)`, except that it impersonates a sender, i.e. the callee sees a sender different from the real caller. To prevent collisions with other deployed contract or externally owned accounts, the impersonated sender address is derived from the real caller address and a salt.
```
Per EIP-1, we include an opcodes hex value for clarity on it's first use in a document.